src/models/db/queries/requests.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../index");
+
+const add = (newRequest) => {
+  return db.oneOrNone(`
+    INSERT INTO
+      requests (username, type_of_timeoff, date_off, time_start, time_end)
```

Comment:
@SomayaB Can you please rename the table to `timeoff_requests`. `Request` is too overloaded a term.

Reply:
@punitrathore do you want me to rename the `requests.js` files too?